Assets/MRTK/Providers/Oculus/XRSDK/MRTK.ThirdParty/MRTK-Quest/Scripts/Input/DeviceManager/Controllers/OculusQuestHand.cs

```diff
@@ -0,0 +1,595 @@
+﻿//------------------------------------------------------------------------------ -
+//MRTK - Quest
+//https ://github.com/provencher/MRTK-Quest
+//------------------------------------------------------------------------------ -
+//
+//MIT License
+//
+//Copyright(c) 2020 Eric Provencher
+//
+//Permission is hereby granted, free of charge, to any person obtaining a copy
+//of this software and associated documentation files(the "Software"), to deal
+//in the Software without restriction, including without limitation the rights
+//to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//copies of the Software, and to permit persons to whom the Software is
+//furnished to do so, subject to the following conditions :
+//
+//The above copyright notice and this permission notice shall be included in all
+//copies or substantial portions of the Software.
+//
+//THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+//AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//SOFTWARE.
+//------------------------------------------------------------------------------ -
+
+using Microsoft.MixedReality.Toolkit;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+
+#if OCULUSINTEGRATION_PRESENT
+using static OVRSkeleton;
+#endif
+
+using Object = UnityEngine.Object;
+//using TeleportPointer = Microsoft.MixedReality.Toolkit.Teleport.TeleportPointer;
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.Oculus
+{
+    /// <summary>
+    /// Oculus Integration Asset package implementation of Oculus Quest articulated hands.
+    /// </summary>
+    [MixedRealityController(
+        SupportedControllerType.ArticulatedHand,
+        new[] { Handedness.Left, Handedness.Right })]
+    public class OculusQuestHand : BaseHand, IMixedRealityHand
```

Comment:
Do we want to make this more generic, like `OculusHand`? I assume in the future, Oculus will ship different headsets with different names that also have hand tracking, and it feels like this code would work there too and isn't specific to Quest.